packages/mui-material/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -306,6 +338,8 @@ const SwipeableDrawer = React.forwardRef(function SwipeableDrawer(inProps, ref)
       return;
     }
 
+    startMaybeSwiping();
+
```

Comment:
```suggestion
```

This looks unrelated to the changes, it can create some changes in the behavior.

Reply:
That was to fix the problem listed here:
https://github.com/mui/material-ui/pull/30759#issuecomment-1074187863

Reply:
This looks like the only blocker at this moment.

Reply:
I've pushed the change.

Reply:
We need a test for this. This component is already loaded, so if we decide to refactor in the future, we may remove this without a test.

Reply:
Or even better, let's create an issue + a separate fix for it, as it doesn't seem much related to these changes.

Reply:
It's here because it was only affecting interactivity with components (such as buttons) inside a swipeable edge drawer.
It's an issue that only affects the contents of the drawer, which previously, were not interactable.
I agree that it could be split out, however to test it, you would need the changes from this PR to be able to interact with the contents of the drawer & trigger the drawer simultaneously, which is where the issue is triggered.

Reply:
Ok, we can then either create a follow up PR, or add the test now. Whatever works better for you :)